### PR TITLE
#677

### LINKED ISSUE #677: bytes.and and bytes.or must accept vararg
At the moment only a single argument can be applied to `bytes.and` and `bytes.or`. Let's make it possible to apply vararg, like it works for `bool.and` and `bool.or`.

### PR DESCRIPTION
**Pull request by kerelape**
Fix #677

### CODE REVIEW

File: eo-runtime/src/main/java/EOorg/EOeolang/EObytes$EOand.java [hunk 1]
```diff
@@ -27,7 +27,9 @@
 import java.math.BigInteger;
 import org.eolang.AtComposite;
 import org.eolang.AtFree;
```
Comment (mximp) [inline eo-runtime/src/main/java/EOorg/EOeolang/EObytes$EOand.java:29]: Unused import

**Review by mximp — COMMENTED** [review]
@kerelape please check comment

File: eo-runtime/src/main/eo/org/eolang/bytes.eo [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 +package org.eolang
 +home https://github.com/objectionary/eo
-+version 0.0.0
++version 0.1.0
```
Comment (mximp) [inline eo-runtime/src/main/eo/org/eolang/bytes.eo:25]: The policy is not to update the version manually.
It should always be `0.0.0` in master and development branches.
It's incremented automatically during deployment process.

**Review by mximp — COMMENTED** [review]
@kerelape please check version comment

**Comment by mximp** [comment]
@yegor256 please merge the PR.

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**